Monika After Story/game/script-topics.rpy

```diff
@@ -5386,7 +5386,7 @@ label monika_timeconcern_day:
     if not persistent._mas_timeconernclose:
         if 6 <= persistent._mas_timeconern <=8:
```

Comment:
This one also has `_mas_timeconern` so that it needs to be fixed too.
I think same thing applies to the persistent `_mas_timeconernclose` in the line above this one.